x-pack/test/security_solution_cypress/cypress/tasks/exceptions.ts

```diff
@@ -183,6 +185,12 @@ export const validateEmptyExceptionConditionField = () => {
 };
 export const submitNewExceptionItem = () => {
   cy.get(CONFIRM_BTN).should('exist');
+  cy.root().then(($page) => {
```

Comment:
Why was it added?

Reply:
When doing some pair programming and testing with @WafaaNasr to check which indices we should delete for the exceptions-specific tests, sometimes we saw displayed an unrelated toaster error message unrelated with the testing we were performing. That toaster was blocking the confirm button we had to click, using force true would solve the issue, but I'm against the use of it.

If you check the current code on `tasks/exceptions.ts`, there are some tests that use the `closeErrorToast()` method to close error toasters before continuing with the interactions with the page. That is why I decided to add a smaller piece of code that checks if a toaster is displayed and if so, close it to continue with the test.

Reply:
@MadameSheema thank you for the explanation. Did I get it right the toaster doesn't always appear? May it's some UI related issue.

Could you add a comment in. the code to clarify your changes?